docs/experiment/guides/debugging-sample-ratio-mismatch.md

```diff
@@ -0,0 +1,89 @@
+
+### 4. How much [variant jumping](// TODO AKHIL link to Brian's article) (same user seeing both treatment and control) was there? Is it more than 5%? 
+
+The issue with variant jumping is that you do not know which variant to attribute the metric to. If a users sees control and then treatment and then does the conversion event, did the user do the conversion event because of the treatment or did the user just need time to think about if they wanted to buy the item and the treatment had no effect on their probability of purchase. You can look at variant jumping chart on the Monitor tab of an experiment. If there is variant jumping, is it because of anonymous users (people constantly logging in and out) or changing device ids? You can see this by looking at the [User Stream](https://help.amplitude.com/hc/en-us/articles/229313067-Look-up-event-data-for-individual-users). 
+
+### 5. Do you see a big difference in the conversion percentage between variants for the Assignment event -> Exposure event funnel? 
+
+Plug in conversion rates and sample size in [here](https://www.socscistatistics.com/tests/ztest/default2.aspx) and see if its statistically significant or not at the 95% significance level, You can find this graph in the Monitor tab of an Experiment. The conversion rates between variants should be similar (within randomness) because the assignment event should randomly split users into two equal cohorts. Since exposure event gets sent right before the user experiences a variant, there should be no difference between users since they have the same user experience from the time the assignment event is sent ot the time the exposure event is sent. 
+
+### 6. Adding a new variant in the middle of an experiment
+
+This is just a special case of #2.
+
+### 7. Removing a variant in the middle of an experiment
+
+This is just a special case of #2.
+
+### 8. Was sticky bucketing turned on and off in the middle of the experiment?
+
+Generally, once you start an experiment you should not change anything.
+
+### 9. Are there a lot of users in the ["Individual Users Allocation"](../general/evaluation/implementation.md#Individual-inclusions) section?
+
+This can cause an SRM because you are not randomly allocating those users. It is fine to add < 1% (// TODO Akhil do I want to be this prescriptive with a number) of the users in the experiment in the "Individual Users Allocation". 
```

Comment:
I don't really want to give a number but instead just say if you have a lot of users. Is it okay to be not precise with the definition of "a lot"?